rinex-cli/README.md

```diff
@@ -47,14 +47,37 @@ if you know how to operate the preprocessing toolkit
 - [quality check](doc/qc.md): RINEX data quality analysis (mainly statistics and only on OBS RINEX at the moment)
 - other advanced operations are documented in the [processing](doc/processing.md) suite
 
-## NAV data special case
+## Data analysis 
 
-When Navigation data are provided, either as primary file, or as Observation enhancer,
-Sv orbit and Kepler information are used to enhance or allow further operations.
+This tool can perform several data analysis. Feasible operations
+are highly dependent on the context provided by the user.
 
-Also, a skyplot view is then provided in the main analysis:
+Based on the type of primary RINEX provided, the main report structure
+will differ: 
 
-<img align="center" width="450" src="https://github.com/georust/rinex/blob/kepler/doc/plots/skyplot.png">
+- OBS RINEX: all observed physics are depicted.
+See this SSI analysis as an example, which is only one type of physics present
+in OBS RINEX:
+
+<img align="center" width="450" src="https://github.com/georust/rinex/tree/main/doc/plots/esbc00dnk_ssi.png">
+
+- NAV RINEX: satellite vehicle clock data is depicted (bias and drift)
+
+<img align="center" width="450" src="https://github.com/georust/rinex/tree/main/doc/plots/sv_clocks.png">
```

Comment:
Is this link broken?

Reply:
nope, it will unlock when this is merged

Reply:
i'm not sure what's best to manage those cases

Reply:
I don't see `sv_clocks.png` on this branch either, but okay, maybe I'm missing something :-).

Reply:
oops correct. *.png are not allowed to be submitted by default, forgot to bypass and introduce this new one